docs/en/core-services/Caching.md

```diff
@@ -205,6 +211,46 @@ A typical configuration will have three tiers for Memory, SSD, and HDD.
 `alluxio.worker.tieredstore.level{x}.dirs.path`.
 
+### Block Allocation Policies
+
+Alluxio uses block allocation policies to define how to allocate new blocks across multiple storage directories (in the same tier or different tiers).
+The allocation policy defines which storage directory to allocate the new block in. 
+
+This is configured by worker property `alluxio.worker.allocator.class`. Out-of-the-box implementations include:
+
+- **MaxFreeAllocator**: Start trying from tier 0 to the lowest tier, try to allocate the block to the storage directory
+that currently has the most availability. **This is the default behavior.**
+
+- **RoundRobinAllocator**: Start trying from tier 0 to the lowest tier. On each tier, maintain a Round Robin order
+of storage directories. Try to allocate the new block into a directory following the Round Robin order, and if that 
+does not work, go to the next lower tier.
+
+- **GreedyAllocator**: This is an example implementation of the `Allocator` interface. It loops from the top tier 
+to the lowest tier, trying to put the new block into the first directory that can contain the block.
+
+### [Experimental] Block Allocation Review Policies
+
+This is an experimental feature added in Alluxio 2.4.1. The interface is subject to change in future versions.
+
+Alluxio uses block allocation review policies to complement allocation policies. In comparison to allocation policies
+which define what the allocation should be, the allocation review process validates allocation decisions and prevent 
+the ones that are not good enough. The `Reviewer` works together with the `Allocator`
+
+This is configured by worker property `alluxio.worker.reviewer.class`. Out-of-the-box implementations include:
+
+- **ProbabilisticBufferReviewer**: Based on the available space in each storage directory, rejects the attempts to put
+new blocks into this directory probabilistically. The probability is determined by `alluxio.worker.reviewer.probabilistic.hardlimit.bytes`
+and `alluxio.worker.reviewer.probabilistic.softlimit.bytes`. When the available space in the directory is under
+`alluxio.worker.reviewer.probabilistic.hardlimit.bytes`, which is `64MB` by default, new blocks will be rejected.
+When the available space in the directory is above `alluxio.worker.reviewer.probabilistic.softlimit.bytes`, which is
+`256MB` by default, new blocks will NOT be rejected. When the available space is between these two values, the probability
+of accepting a new block goes down linearly to as the availability goes down. We choose to reject new blocks early before
+a directory is filled up, because the existing blocks in the directory can expand in size as we read new data in the block.
+Leaving a buffer in each directory can reduce the chance of eviction. **This is the default behavior.**
+
+- **AcceptingReviewer**: This reviewer accepts every block allocation. So the behavior will be exactly the same
```

Comment:
@gpang Do we want to mention the tier allocation issue if they use this AcceptingReviewer?

Reply:
No, I don't think that is necessary.